components/raftstore/src/store/snap.rs

```diff
@@ -1492,6 +1492,11 @@ impl SnapManager {
         path.to_str().unwrap().to_string()
     }
 
+    pub fn get_path_for_tablet_checkpointer(&self, key: &SnapKey) -> PathBuf {
```

Comment:
minor: checkpointer ---> checkpoint
Maybe the fn can be renamed as get_tablet_checkpoint_path(..)

Reply:
I follower `checkpointer`  with https://github.com/tikv/tikv/pull/13693.